quinn-proto/src/config.rs

```diff
@@ -829,8 +830,8 @@ impl ServerConfig {
     ///
     /// Uses a randomized handshake token key.
     pub fn with_single_cert(
-        cert_chain: Vec<rustls::Certificate>,
-        key: rustls::PrivateKey,
+        cert_chain: Vec<CertificateDer<'static>>,
+        key: PrivateKeyDer<'static>,
```

Comment:
How do we know `rustls-pki-types` is enabled?

Reply:
Because `feature = "rustls"` will now also enable `feature = "rustls-pki-types"`.